### DISCUSSION TITLE
Guidance with supplementing auto-generated help.

### DISCUSSION THREAD

**Discussion by jcasale** [spectreconsole/spectre.console#897] [General]
I need to append some text to the auto-generated help to provide my users with some additional direction that is not related to any specific command. For example, my utility logs to the Windows application event log and I want to outline all the events and their IDs.

Is it possible to accomplish this?

**Comment by lonix1** [comment]
Did you find out how to do that? I'm also interested in that.

**Comment by jcasale** [comment]
You can't, and so much of the library is restricted through access modifiers that you can't even override the behavior.

**Reply by lonix1** [reply]
Yes I've discovered that myself, many of the things I'm interested in are `internal`. Another interesting [related thread](https://github.com/spectreconsole/spectre.console/discussions/643). 

Nils commented:

> I would suggest to open individual discussions on the ideas you have, so each idea can be discussed individually.
